src/components/footer/footer.jsx

```diff
@@ -0,0 +1,22 @@
+// import react from "react";
+
+export const Footer = () => {
+  return (
+    <div class="footer-home flex-col">
+      <h1 class="h3">Created by Yogesh Gupta</h1>
+      <div class="social" id="social">
+        <a href="https://github.com/yogesh-">
```

Comment:
you can also return this data by using - constant file . make array of object eg.  let data = [  {img:"../path", link:"http://" , name:"facebook" } , {.....} ,{.....}]    fetch -> data.map((item,index)=>{ return ( <li> {item.img}  <li> ) }) ... so on

Reply:
okay,